### DISCUSSION TITLE
Does CustomTkinter includes any font? Or it loads a font from our computer?

### DISCUSSION THREAD

**Discussion by CarlosVSnakieProgrammer** [TomSchimansky/CustomTkinter#254] [Q&A]
Hello, I'd like to now this because i want to share my app and include all the necesary license texts (like MIT python modules that i use, atributtions, resources, etc). And seems that CustomTkinter use Roboto (wich I think that it use Apache 2.0). So in that case I probably have to Include the Apache 2.0 license of the font in my app.

Thank you for answering me.
And thanks for sharing us this awesome module :)

**Comment by felipetesc** [comment]
It include fonts, they are inside CustomTkinter/customtkinter/assets/fonts, you can see it at https://github.com/TomSchimansky/CustomTkinter/tree/master/customtkinter/assets/fonts. You can load external fonts editing your code

**Reply by CarlosVSnakieProgrammer** [reply]
Thanks I din't notice it!

**Comment by TomSchimansky** [comment]
On Windows and Linux CustomTkinter loads Roboto font from .otf files and uses them as default font. On macOS CustomTkinter uses SF Display as default font which is standard on macOS at the moment. I downloaded the Roboto .otf files from Google fonts.

**Reply by CarlosVSnakieProgrammer** [reply]
Thank you for answering! :)
